pkg/svcat/service-catalog/instance.go

```diff
@@ -165,23 +165,41 @@ func (sdk *SDK) InstanceToServiceClassAndPlan(instance *v1beta1.ServiceInstance,
 }
 
 // Provision creates an instance of a service class and plan.
-func (sdk *SDK) Provision(instanceName, className, planName string, opts *ProvisionOptions) (*v1beta1.ServiceInstance, error) {
-	request := &v1beta1.ServiceInstance{
-		ObjectMeta: v1.ObjectMeta{
-			Name:      instanceName,
-			Namespace: opts.Namespace,
-		},
-		Spec: v1beta1.ServiceInstanceSpec{
-			ExternalID: opts.ExternalID,
-			PlanReference: v1beta1.PlanReference{
-				ClusterServiceClassExternalName: className,
-				ClusterServicePlanExternalName:  planName,
+func (sdk *SDK) Provision(instanceName, classKubeName, planKubeName string, provisionClusterInstance bool, opts *ProvisionOptions) (*v1beta1.ServiceInstance, error) {
```

Comment:
this function changed because of the mentioned comment I'd like added.

It's always kube names that come in now.